.rubocop.yml

```diff
@@ -24,7 +24,7 @@ Metrics/MethodLength:
 Style/AccessorMethodName:
   Enabled: false
 
-Style/MethodCallParentheses:
+Style/MethodCallWithoutArgsParentheses:
```

Comment:
I'm conflicted about this one. On the one hand, you really want to be able to call predicates without parenthesis (e.g. `return if foo.bar?`). On the other hand, I spent some time debugging a ticket because of a function returning a hash which was called without parenthesis (`foo['key'] = 'value'` with foo being a function), because I did not know that `foo` was a function. In that case, parenthesis should have been enforced.

If rubocop can make parenthesis mandatories for non-predicates functions, then I think it should be enabled.